src/Stratis.Bitcoin.IntegrationTests.Common/Runners/StratisBitcoinPosRunner.cs

```diff
@@ -17,16 +17,22 @@ namespace Stratis.Bitcoin.IntegrationTests.Common.Runners
 {
     public sealed class StratisBitcoinPosRunner : NodeRunner
     {
-        public StratisBitcoinPosRunner(string dataDir, Network network, string agent = "StratisBitcoin")
+        private readonly bool isGateway;
+
+        public StratisBitcoinPosRunner(string dataDir, Network network, string agent = "StratisBitcoin", bool isGateway = false)
             : base(dataDir, agent)
         {
             this.Network = network;
+            this.isGateway = isGateway;
         }
 
         public override void BuildNode()
         {
             var settings = new NodeSettings(this.Network, ProtocolVersion.PROVEN_HEADER_VERSION, this.Agent, args: new string[] { "-conf=stratis.conf", "-datadir=" + this.DataFolder });
 
+            if (this.isGateway)
```

Comment:
I'm not sure if this will have much effect, because the earliest protocol version of both X and SBFN 'in the wild' is `ALT_PROTOCOL_VERSION` (70000) anyway.

Reply:
Ah okay, I did it to keep it consistent with the other tests which were using a feral custom node. 

Given this is using PROVEN_HEADERS though and it's trying to connect to a StratisX, doesn't it need this?

Reply:
Ah yes, in the back of my mind I thought the fallback to 70000 as the minimum was done elsewhere, but you are correct, it needs to be in the node creation.